pkg/agent/multicluster/noderoute/controller_common.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2022 Antrea Authors
```

Comment:
Do we need a sub-package "noderoute"? I feel we can put all files in multicluster/.

Reply:
done.